src/pds/registry/utils/treks/create_treks_pds4.py

```diff
@@ -0,0 +1,166 @@
+    valid_targets = [
+        # planets
+        "mars",
+        "mercury",
+        "venus",
+        # moons
+        "moon",
+        "phobos",
+        "titan",
+        # asteroids
+        "ceres",
+        "ryugu",
+        "vesta",
+        # run script on all targets
+        "all"
+    ]
+
+    # set up command line args
+    parser = argparse.ArgumentParser(description="Create and save pds4 xml labels created for Treks",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("-s",
+                        "--save-xml",
+                        action="store_true",
+                        help="True if you want the xml files to be saved",
+                        default=True)
+    parser.add_argument("-d",
+                        "--destination-directory",
+                        help="Directory to save pds4 xml files",
+                        default="treks_xml")
+    parser.add_argument("-v",
+                        "--verbose",
+                        action="store_true",
+                        help="Verbosity of file creation",
+                        default=False)
+    parser.add_argument("-l",
+                        "--save-logs",
+                        action="store_true",
+                        help="Save logs of run",
+                        default=False)
+    parser.add_argument("-t",
+                        "--target",
+                        help=f"Treks target to generate labels for: {valid_targets}",
+                        default="titan")
+
+    args = parser.parse_args()
+
+    # parse args
+    save_xml = args.save_xml
+    dest = args.destination_directory
+    verbose = args.verbose
+    save_logs = args.save_logs
+    target = args.target.lower()
+
+    # remove whitespace from target
+    target = target.strip()
+    target = target.replace(" ", "")
+
+    selected_targets = []
+    if target == 'all':
+        selected_targets = valid_targets[:-1]
+    else:
+        selected_targets = [target]
+
+    # initialize logger
+    if save_logs:
+        logging.basicConfig(filename='create-treks-pds4-log.log', level=logging.WARNING)
+        logging.warning("target,product_label,missing_data_location,tag")
+
+    total_start = time.time()
+    for target in selected_targets:
+        target_dest = dest + "/" + target
+
+        if verbose:
+            print("Creating pds4 xml for target:", target)
+
+        if save_xml:
+            # create destination path if it does not exist
+            Path(target_dest).mkdir(parents=True, exist_ok=True)
+
+        # get json from url
+        url = "https://trek.nasa.gov/" + target + "/TrekServices/ws/index/eq/" + \
+            "listVisibleLayers?proj=urn:ogc:def:crs:EPSG::60620&start=0&rows=2147483647"
+        response = requests.get(url, timeout=30)
+        json_data = response.json()
+
+        # get data from response
+        # num_found = json_data["response"]["numFound"]
```

Comment:
Whenever commenting-out code, either:

- Add a comment explaining _why_ it's commented out
- If you can't come up with such a comment, then nuke it completely

This helps prevent future readers of the code from wondering if a certain line should be put back in or not for some reason or another

Reply:
shoot I missed that while cleaning everything! I'll make sure to change that.